BB2-3298: Added access_grant_expiration to token response (#370)
CMSgov/bluebutton-site-static: _pages/developers.md

##### Token response
~~~
{
    "access_token": "oQlduHNr09GKCU506GOgp8OarrAy2q",
    "expires_in": 16768.523842,
    "token_type": "Bearer",
    "scope": "profile patient/Patient.read patient/ExplanationOfBenefit.read patient/Coverage.read",
    "refresh_token": "wDimPGoA8vwXP51kie71vpsy9l17HN",
    "access_grant_expiration": "2025-09-05 19:17:53Z"
}
~~~
Applications in the "10 hours" access category do not receive a refresh token in the BB2.0 token response.

> Note that for applicants in the 10 hours or 13 months access expiration category, you will also see a response that shows the access_grant_expiration date for that user.

Reviewer: ```suggestion
Note that for applications in the "10 hours" or "13 months" access expiration categories, the response includes the `access_grant_expiration` date for that user.
```
- I think there was a typo in the original suggestion of `applicants` instead of `applications`.
- We should be consistent about how we show the expiration categories, and currently `"10 hours"` seems to be the standard. I might recommend `**10 hours**` instead, which would show in bold on the page, but whatever we decide, we want to be consistent within the page.
- I switched up the wording slightly to take out the future tense, but this is optional and minor.

@jtheil-nava can you provide some feedback on the above?
